presidio-analyzer/tests/test_us_driver_license_recognizer.py

```diff
@@ -76,28 +59,21 @@ def test_invalid_us_driver_license(self):
     # Driver License - Digits (very weak) - 0.05
     # Regex: r'\b([0-9]{1,9}|[0-9]{4,10}|[0-9]{6,10}|[0-9]{1,12}|[0-9]{12,14}|[0-9]{16})\b'
 
-    # TODO: enable with task #582 re-support context model in analyzer
-    # def test_valid_us_driver_license_very_weak_digits(self):
-    #     num = '123456789 1234567890 12345679012 123456790123 1234567901234'
-    #     results = us_license_recognizer.analyze(num, entities)
-    #
-    #     assert len(results) == 5
-    #     for result in results:
-    #         assert 0 < result.score < 0.1
-
-    # def test_valid_us_driver_license_very_weak_digits_exact_context(self):
-    #     num = '1234567901234'
-    #     context = 'my driver license is: '
-    #     results = us_license_recognizer.analyze(context + num, entities)
-    #
-    #     assert len(results) == 1
-    #     assert 0.55 < results[0].score < 0.91
-    # def test_load_from_file(self):
-    #     path = os.path.dirname(__file__) + '/data/demo.txt'
-    #     text_file = open(path, 'r')
-    #     text = text_file.read()
-    #     results = us_license_recognizer.analyze(text, entities)
-    #     assert len(results) == 1
+    def test_valid_us_driver_license_very_weak_digits(self):
+        num = '123456789 1234567890 12345679012 123456790123 1234567901234'
+        results = us_license_recognizer.analyze(num, entities)
+
+        assert len(results) == 5
+        for result in results:
+            assert 0 < result.score < 0.02
+
+    def test_load_from_file(self):
```

Comment:
This is a setup method or a fixture and should be placed in the correct place

Reply:
👍